sympy/physics/mechanics/tests/test_linearize.py

```diff
@@ -296,6 +331,13 @@ def test_linearize_pendulum_lagrange_nonminimal():
     assert _simplify_matrix(A) == Matrix([[0, 1], [-9.8/L, 0]])
     assert B == Matrix([])
 
+    # Check an alternative solver
+    A, B, inp_vec = LM.linearize([q2], [q2d], [q1], [q1d], op_point=op_point,
```

Comment:
This is just the same solver, only now directly calling `LUsolve`, instead of using `solve` with `'LU'` as argument.

Reply:
Yes, I think I just added some checks to make sure the new kwarg worked at all, not trying to use a different solver but simply using the kwarg.